### ISSUE TITLE
Unable to determine if aria-controls referenced ID exists on the page while using aria-haspopup: aria-controls

### ISSUE THREAD

**Issue by audiofrequenz** [primefaces/primevue#8092]
### Describe the bug

Storybook and accessibility extension mentions following errors:
ARIA attribute values valid
aria-valid-attr-value
Critical
1
Use only valid values for ARIA attributes (no typos or invalid values). [Learn how to resolve this violation]
(https://dequeuniversity.com/rules/axe/4.10/aria-valid-attr-value?application=axeAPI)
**Unable to determine if aria-controls referenced ID exists on the page while using aria-haspopup: aria-controls="pv_id_2_panel"**

I think the same problem seems to be fixed inside the PrimeNG Project for dropdown.

Here is the existing Ticket and Fix:

> The aria-control attribute references an element that doesn't exist as it's dynamically created when the user clicks the dropdown icon.
> A potential solution is to dynamically add the attribute when the element is showing.
> https://github.com/primefaces/primeng/issues/14851 and https://github.com/psy-repos-angular/primeng/commit/9173e950cfa561dccc74f8a5b6b6282c659de8ba

<img width="1611" height="543" alt="Image" src="https://github.com/user-attachments/assets/b42e8f58-ba04-44aa-bf26-645661dcc7d4" />

<img width="1612" height="496" alt="Image" src="https://github.com/user-attachments/assets/37051dc8-3702-46e8-9f8c-f202be4ac8b6" />

But its a problem in at least following components:

1. Datepicker
2. MultiSelect
3. Select

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-hqetbfdt?file=src%2FApp.vue

### Environment

Win 11 with a local version of storybook and storybook/addon-a11y for accessibility tests

### Vue version

3.5.21

### PrimeVue version

4.3.9

### Node version

22.14.0

### Browser(s)

Chrome 140.0.7339.81

### Steps to reproduce the behavior

1. Create Project
2. Include Datepicker / MultiSelect / Select with labels
3. Use storybook with storybook/addon-a11y
4. Take a look in storybook in the components accessibility

### Expected behavior

There should not be following error inside the accessibility

ARIA attribute values valid
aria-valid-attr-value
Critical
1
Use only valid values for ARIA attributes (no typos or invalid values). [Learn how to resolve this violation](https://dequeuniversity.com/rules/axe/4.10/aria-valid-attr-value?application=axeAPI)

**Comment by geiregjo** [comment]
I created a [pull request](https://github.com/primefaces/primevue/pull/8162) for this issue.
